pipelines/A15/metadata_table_columns.csv

```diff
@@ -0,0 +1,5 @@
+name,table_name,data_type,description
```

Comment:
@cmdoret I have added a column `example_values` to the `metadata_table_columns` table, see here: can you please also add this column in your csv file?